lib/src/components/NxTextInput/NxTextInput.tsx

```diff
@@ -61,19 +63,26 @@ const NxTextInput = forwardRef<HTMLInputElement | HTMLTextAreaElement, Props>(
       }
 
       return (
-        <NxTooltip open={isInvalid}
-                   title={firstValidationError}
-                   className="nx-tooltip--validation-error"
-                   placement="top-end">
-          {React.createElement(element, {
-            ...newProps,
-            ref,
-            type: typeAttr,
-            onChange: inputOnChange,
-            className: internalClassName,
-            onKeyPress: inputOnKeyPress
-          })}
-        </NxTooltip>
+        <div className={internalClassName}>
+          <div className="nx-text-input__box">
+            {React.createElement(element, {
+              ...newProps,
+              disabled,
+              ref,
+              type: typeAttr,
+              onChange: inputOnChange,
+              className: "nx-text-input__input",
+              onKeyPress: inputOnKeyPress
+            })}
+            <span className="nx-text-input__validation-icon">
```

Comment:
What's the purpose of this wrapper?

Reply:
> What's the purpose of this wrapper?

Because we're using flexbox the icons are seen as two separate columns if they aren't wrapped